base/atomics.jl

```diff
@@ -88,10 +108,21 @@ function atomic_cas! end
 
 Atomically exchange the value in `x`
 
-Atomically exchanges the value in `x` with `newval`. Returns the old
+Atomically exchanges the value in `x` with `newval`. Returns the old (!)
```

Comment:
Done in https://github.com/JuliaLang/julia/pull/20412/commits/dd3b5d74ebc89ce22041740ca149e43e757ba196